backend/app/api/question.py

```
@@ -0,0 +1,155 @@
+    if quiz.owner_id != current_user.id:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN, detail="Not authorized to delete this question"
+        )
+
+    await question_services.remove_question(session=session, question_id=question_id)
+    return None
+
+
+@router.get("/", response_model=list[QuestionRead])
+async def read_all_questions(
+    quiz_id: int, session: AsyncSession = Depends(get_db)
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)